### PR TITLE
 Create New offer#20

### PR DESCRIPTION
**Pull request by Quelqun2k4**
Dear @Dine16 and @GaetanRole ,

For this pull request : 

- Create the new offer's page,
- Add a new scss file "offer" and a datepicker in js => So i add two entry on webpack,
- Create a new Service for calculate fees and add it to the offer controller in "newAction",
- Check all AppBundle for Offers ( phpcs and cleaning all useless informations).
- Check W3C validator : there is just a warning about Date Type which can be not supported by all browsers i had to add polyfill but want to look at it before).

### CODE REVIEW

File: app/Resources/views/offer/new.html.twig [hunk 1]
```diff
@@ -1,51 +1,151 @@
...
 
-                <div class="form-group">
-                    {{ form_label(form.partnershipNumber) }}
-                    {{ form_widget(form.partnershipNumber) }}
-                    {{ form_errors(form.partnershipNumber) }}
+{% block body %}
+    <section>
+        <!-- Form to create your profile -->
+        <div class="container-fluid light-blue-subtitle-text mt-50">
+            <div class="row boxing-canvas mt-40 " >
```
Comment (Dine16) [inline app/Resources/views/offer/new.html.twig]: un espace en trop

File: app/Resources/views/offer/new.html.twig [hunk 2]
```diff
@@ -1,51 +1,151 @@
...
+            <div class="row text-center boxing-canvas">
+                <div class="col-md-12">
+                    <table class="table table-striped table-hover table-responsive">
+                        <thead>
+                            <tr>
+                                <th>Libellé</th>
+                                <th>Montant Brut</th>
+                                <th>Frais de service</th>
+                                <th>Fonds Brut</th>
+                                <th>Nb de partenaires</th>
```
Comment (Dine16) [inline app/Resources/views/offer/new.html.twig]: Nombre de partenaires

File: app/Resources/views/offer/new.html.twig [hunk 3]
```diff
@@ -1,51 +1,151 @@
...
+            </div>
+            <div class="row text-center boxing-canvas">
+                <div class="col-md-12">
+                    <table class="table table-striped table-hover table-responsive">
+                        <thead>
+                            <tr>
+                                <th>Libellé</th>
+                                <th>Montant Brut</th>
+                                <th>Frais de service</th>
+                                <th>Fonds Brut</th>
```
Comment (Dine16) [inline app/Resources/views/offer/new.html.twig]: Fonds bruts

File: app/Resources/views/offer/new.html.twig [hunk 4]
```diff
@@ -1,51 +1,151 @@
...
-    <ul>
-        <li>
-            <a href="{{ path('dashboard_index') }}">Revenir au Dashboard</a>
-        </li>
-    </ul>
+        <div class="container-fluid mt-20">
+            <div class="row boxing-canvas mt-20">
+                <div class="col-md-10 col-md-offset-1 mt-20">
+                    <div class="col-md-12 text-center mt-20">
+                        <i class="fas fa-hand-holding-usd  degradeblue" style="font-size: 50px"></i>
```
Comment (Dine16) [inline app/Resources/views/offer/new.html.twig]: un espace en trop

File: app/Resources/views/offer/new.html.twig [hunk 5]
```diff
@@ -1,51 +1,151 @@
...
+                                {{ form_widget(form.partnershipNumber, {'attr': {'class':'form-control input-field', 'placeholder' : "Nombre de participant(s)" }}) }}
+                                {{ form_errors(form.partnershipNumber) }}
+                            </div>
+                        </div>
+                        <div class="col-md-6 text-center box-border mt-30">
+                            <p class="bold-text pt-20">Calcul du montant des commissions retenues :</p>
+                            <table class="table table-striped table-hover">
+                                <thead>
+                                <tr>
+                                    <th>Taux de commissions</th>
```
Comment (Dine16) [inline app/Resources/views/offer/new.html.twig]: Taux de commission sans s non ?

File: app/Resources/views/offer/new.html.twig [hunk 6]
```diff
@@ -1,51 +1,151 @@
...
+                                </tr>
+                                </thead>
+                                <tbody>
+                                <tr>
+                                    <td>15 %</td>
+                                    <td>200 €  $lt 10 000 €</td>
+                                </tr>
+                                <tr>
+                                    <td>10 %</td>
+                                    <td>10 001 € &lt 50 000 €</td>
```
Comment (Dine16) [inline app/Resources/views/offer/new.html.twig]: mettre "et" plutôt que "&" comme à la ligne en dessous

File: app/Resources/views/offer/new.html.twig [hunk 7]
```diff
@@ -1,51 +1,151 @@
...
+                        <input class="col-sm-12 col-xs-12 btn btn-primary backblue-btn mt-20" type="submit" value="{{ "Enregistrer"|trans }}" />
+                    </div>
+                    {{ form_end(form) }}
+                </div>
+            </div>
+            <div class="row subtitle-boxing-canvas">
+                <!--Return to Dashboard-->
+                <div class="col-sm-12 back-dashboard normal-subtitle-text">
+                    <i class="fas fa-reply"></i>&nbsp;
+                    <a href="{{ path('dashboard_index') }}" class="five-color"> Retourner sur votre tableau de bord ?</a>
```
Comment (Dine16) [inline app/Resources/views/offer/new.html.twig]: un espace en trop

File: app/Resources/views/offer/new.html.twig (hunk 1 above)
Comment (Quelqun2k4) [reply app/Resources/views/offer/new.html.twig]: VU

File: app/Resources/views/offer/new.html.twig (hunk 3 above)
Comment (Quelqun2k4) [reply app/Resources/views/offer/new.html.twig]: Vu

File: app/Resources/views/offer/new.html.twig (hunk 4 above)
Comment (Quelqun2k4) [reply app/Resources/views/offer/new.html.twig]: vu

File: app/Resources/views/offer/new.html.twig (hunk 5 above)
Comment (Quelqun2k4) [reply app/Resources/views/offer/new.html.twig]: vu

File: app/Resources/views/offer/new.html.twig (hunk 6 above)
Comment (Quelqun2k4) [reply app/Resources/views/offer/new.html.twig]: Il s'agit  d' appellations pour le signe "<"  c'est une recommandation de W3C validator.

File: app/Resources/views/offer/new.html.twig (hunk 7 above)
Comment (Quelqun2k4) [reply app/Resources/views/offer/new.html.twig]: vu

File: app/Resources/views/offer/new.html.twig (hunk 2 above)
Comment (Quelqun2k4) [reply app/Resources/views/offer/new.html.twig]: vu

**Pull request closed without merging**